hrcase/hrcase.php

```diff
@@ -340,10 +340,15 @@ function hrcase_civicrm_alterContent( &$content, $context, $tplName, &$object )
   }
 }
 
+/**
+ * Implementation of hook_civicrm_post, executed after task creation/update
+ *
+ * @param $op string, the type of operation being performed;
```

Comment:
The convention we use is the same one as Drupal's on which the @param description should be on a newline ([see here](https://www.drupal.org/node/1354#param)). Also, the type should come before the param name, so:

```
@param string $op
  The Type of the operation being performed
```

There's no need for period or semicolon at the end of the description.

By the way, if your IDE/Editor supports coding styles, it might be good to set it to use Drupal's to help you follow the standards.